packages/components/src/visually-hidden/component.js

```diff
@@ -1,8 +1,16 @@
 /**
  * Internal dependencies
  */
-import { createComponent } from '../ui/utils';
-import { useVisuallyHidden } from './hook';
+import { useContextSystem, contextConnect } from '../ui/context';
+import { VisuallyHidden as VisuallyHiddenWrapper } from './styles';
+
+/**
+ * @param {import('../ui/context').PolymorphicComponentProps<{ children: import('react').ReactNode }, 'div'>} props
+ */
+function VisuallyHidden( props ) {
+	const contextProps = useContextSystem( props, 'VisuallyHidden' );
```

Comment:
It doesn't look like the `VisuallyHidden` component was reading from the context system prior to this PR — just making sure this is noted during the review

Reply:
Yes! Because we're now no longer manually creating the classname it has to come from `useContextSystem` 🤷

Reply:
Makes sense!